generate-address-list.py

```diff
@@ -74,6 +86,10 @@ def write_addresses_json(addresses, asset, outpath):
     with open("{}/sanctioned_addresses_{}.json".format(outpath, asset), 'w') as out:
         out.write(json.dumps(addresses, indent=2)+"\n")
 
+def write_addresses_csv(addresses, asset, outpath):
+    with open("{}/sanctioned_addresses_{}.csv".format(outpath, asset), 'w') as out:
+        for address in addresses:
+            out.write(address+";\n")
```

Comment:
The only difference to the TXT file is the `;`. Is a CSV output really needed?

Reply:
of course, it's just  PoC. 
in the future better to put delimeter